src/login/logind-gperf.gperf

```diff
@@ -37,6 +37,7 @@ Login.HandleHibernateKeyLongPress,  config_parse_handle_action,         0, offse
 Login.HandleLidSwitch,              config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch)
 Login.HandleLidSwitchExternalPower, config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch_ep)
 Login.HandleLidSwitchDocked,        config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch_docked)
+Login.HandleSAKSequenceLongPress,   config_parse_handle_action,         0, offsetof(Manager, handle_sak_sequence_long_press)
```

Comment:
why long press? the combo requires a full kbd anyway and is cryptic enough to type, hence there should be no reason to even combine it with long press...

the long press stuff mostly exists so that devices that have no input except for a single power button so that we can derive two distinct actions from that. But i don't really see the need for this for SAK.